COPYING

```diff
@@ -1,14 +1,13 @@
-                  GNU LESSER GENERAL PUBLIC LICENSE
-                       Version 2.1, February 1999
+                  GNU LIBRARY GENERAL PUBLIC LICENSE
```

Comment:
@EmmanuelP: same comment as here: https://github.com/AravisProject/aravis/pull/985/files#r1956751674
this now states "library" instead of "lesser" - i don't think that this was your intention?